test/services/IncorporationInformationServiceSpec.scala

```diff
@@ -98,4 +100,85 @@ class IncorporationInformationServiceSpec extends VatRegSpec with Inspectors wit
         testIncorporationInfo.statusEvent.incorporationDate
     }
   }
+
+  "retrieveSicCodes" must {
```

Comment:
would like to see a test where the response = 200, json is invalid, does this recover correctly?